### PR TITLE
Update the CrateDB Docker image homepage

### PR DESCRIPTION
**Pull request by norosa**
Specifically:

- Update wording to reflect the `crate/crate` `README.rst` file
- Adds some small copyedits
- Adds a link to the screenshot we use

### CODE REVIEW

**Comment by tianon** [comment]
I'll also add a CC to [the `crate` image maintainers](https://github.com/docker-library/official-images/blob/30cd68b60db0ae340832eb691bb1002745c8cdd9/library/crate#L1-L4) for further content review: @mfussenegger @seut @mkleen @amotl

**Comment by norosa** [comment]
@tianon I deleted the old PR because I thought I'd pushed it from my personal fork. perhaps I was mistaken. I will edit the short description

**Comment by norosa** [comment]
@tianon I believe I have resolved the issue now

for some reason, I am having a lot of trouble rebasing locally. I think I borked my client config when I was trying to update our fork. fortunately, the GitHub interface allowed me to fix the "conflicts" that resulted

File: crate/get-help.md [hunk 1]
```diff
@@ -1 +1 @@
-[project documentation](https://crate.io/docs/), [StackOverflow](https://stackoverflow.com/tags/crate), [Slack](https://crate.io/docs/support/slackin/), or [paid support](https://crate.io/pricing/)
+[project documentation](https://crate.io/docs/), [StackOverflow](https://stackoverflow.com/tags/crate), [support channels](https://crate.io/support/)
```
Comment (amotl) [inline crate/get-help.md:1]: In order to align with https://github.com/crate/crate-admin/pull/702, I am humbly asking to update the tag on the link to Stack Overflow.

```suggestion
[project documentation](https://crate.io/docs/), [StackOverflow](https://stackoverflow.com/tags/cratedb), [support channels](https://crate.io/support/)
```

**Review by amotl — CHANGES_REQUESTED** [review]
Thanks a stack! Just a minor copyedit request from my side.

**Comment by amotl** [comment]
Hi Naomi,

I noticed another thing, sorry to bother you once more.

At the ["Try CrateDB"](https://github.com/docker-library/docs/blob/f67955515354dc89b0121c1c224ea19631e6d014/crate/content.md#try-cratedb) section, can you also update the "Getting Started With CrateDB on Docker" link to point to [1] instead of [2]?

With kind regards,
Andreas.

[1] https://crate.io/docs/crate/howtos/en/latest/deployment/containers/docker.html
[2] https://crate.io/docs/crate/guide/en/latest/deployment/containers/docker.html

**Comment by norosa** [comment]
@amotl the original purpose of this pr was to correct that link from [2] to [1] as [1] is the old URL and [2] is the new URL. have I got that right? I think this is good to go

**Comment by norosa** [comment]
@amotl ah. I had some changes I hadn't pushed. and last time I tried, I resolved the conflicts incorrectly. phew! the changes should be complete and correct now. can you recheck please?

**Review by amotl — APPROVED** [review]
Thank you very much for adjusting the content, @norosa. @tianon: Good to go from my side.

**Comment by amotl** [comment]
Thank you, @tianon and @yosifkit.

**Pull request merged**